test/server/api/project/index.project.spec.js

```diff
@@ -1293,6 +1293,97 @@ describe('PROJECT REST API', function () {
                         done();
                     });
             });
+
+            //export tests
+            it('should export project /projects/:ownerId/:projectId/commits/:commitHash/export',
+                function (done) {
+                    var url = server.getUrl() + '/api/projects/' + projectName2APIPath(projectName) + '/commits/' +
+                        importResult.commitHash.substring(1) + '/export';
+                    agent.get(url)
+                        .end(function (err, res) {
+                            expect(res.status).equal(200, err);
```

Comment:
Since the timeout was brought up, we should aim to avoid that at least in new tests... Try catch around the expects.